npm/react/src/createMount.ts

```diff
@@ -1,24 +1,12 @@
 import * as React from 'react'
 import getDisplayName from './getDisplayName'
 import {
-  injectStylesBeforeElement,
   getContainerEl,
   ROOT_SELECTOR,
   setupHooks,
 } from '@cypress/mount-utils'
 import type { InternalMountOptions, MountOptions, MountReturn, UnmountArgs } from './types'
 
-/**
- * Inject custom style text or CSS file or 3rd party style resources
- */
-const injectStyles = (options: MountOptions) => {
-  return (): HTMLElement => {
-    const el = getContainerEl()
-
-    return injectStylesBeforeElement(options, document, el)
```

Comment:
Rather than removing outright, we should check the options for any of the `StyleOptions` properties and throw an error if still in use so that users are more aware of the removal of this functionality

Reply:
https://github.com/cypress-io/cypress/pull/24419/commits/4af263a7534003b00d7c291abf444592b1b1b014

I could not get the for loop style testing working with the Svelte test. You cannot use `expect(() => ).to.throw` for custom commands (like `cy.mount`). You need to do Cypress.on...

I was not able to get this working with a for loop like the others due to how we queue up everything, the `on` listener was overriding each time. Either way, I think we've got good coverage for this error handling.